spk/tvheadend/Makefile

```diff
@@ -35,9 +34,30 @@ POST_STRIP_TARGET = tvheadend_extra_install
 BUSYBOX_CONFIG = usrmng
 ENV += BUSYBOX_CONFIG="$(BUSYBOX_CONFIG)"
 
+include ../../mk/spksrc.common.mk
+
+# Reuse ffmpeg libraries
+# Requires to invoke: make [package|publish] ARCH= TCVERSION=
+export FFMPEG_DIR = $(shell pwd)/../ffmpeg/work-$(ARCH)-$(TCVERSION)/install/var/packages/ffmpeg/target
+
+ifneq ($(wildcard $(FFMPEG_DIR)),)
+$(info Depend on ffmpeg package libraries)
```

Comment:
Please discard "info" and "warning" messages the same way I did... because content is "cat" when generating PLIST and produces error at spksrc "tar" command. So I preferred to removed them to keep make output clean of any error.